web-app/js/portal/cart/Downloader.js

```diff
@@ -129,5 +131,14 @@ Portal.cart.Downloader = Ext.extend(Ext.util.Observable, {
 
     _sanitiseFilename: function(str) {
         return str.replace(/:/g, "#").replace(/[/\\ ]/g, "_");
+    },
+
+    _getServiceMessage: function(response) {
+        if (response) {
+            return response;
+        }
+        else {
+            return "";
+        }
```

Comment:
This can be shortened to just `response ?: ""`. Possibly making this function redundant?